new/STU0008Lex.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="STU0008Lex" xml:lang="en" type="studies">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gǝʿǝz-Latin Lexicon</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>According to <bibl><ptr target="bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange></bibl> 
+                    the author is <persName ref="PRS14230GBRaimondi"/>.
+                    The Vocabulary is apparently taken from poetry and arranged by initial letter. 
```

Comment:
what is taken from poetry? The list of lemmata? definitions? not clear

Reply:
now saw the comment to the MS. So then write 
"The Vocabulary lists terms apparantly encountered in poetic texts"